EIPS/eip-7685.md

```diff
@@ -70,7 +55,7 @@ Extend the header with a new 32 byte value `requests_hash`:
 def compute_requests_hash(list):
     return keccak256(rlp.encode([rlp.encode(req) for req in list]))
 
-block.header.requests_root = compute_requests_hash(block.body.requests)
+block.header.requests_root = compute_requests_hash(requests)
```

Comment:
```suggestion
block.header.requests_hash = compute_requests_hash(requests)
```
There's another use of `"requests_root"` in the "Ordering" section below, I think we should take the opportunity to update that too.

Reply:
fixed. tysm